paf-mvp-operator-client-express/src/operator-client.ts

```diff
@@ -69,4 +89,29 @@ export class OperatorClient {
     );
     return this.getIdsPrefsRequestBuilder.getRedirectUrl(getIdsPrefsRequestJson);
   }
+
+  private createUnsignedSeed(transactionIds: TransactionId[], timestamp = getTimeStampInSec()): UnsignedData<Seed> {
+    return {
+      version: CurrentModelVersion,
+      transaction_ids: transactionIds,
+      publisher: this.clientHost,
+      source: {
+        domain: this.clientHost,
+        timestamp,
+      },
+    };
+  }
+
+  private addSignatureToSeed(unsigned: UnsignedData<Seed>, signature: Signature): Seed {
+    return {
```

Comment:
More concise and more consistent with other methods:

```javascript
return {
  ...unsigned,
  source: {
    ...unsigned.source,
    signature
  }
};
```

Also more maintainable as you wouldn't have to change this method if any attribute is added to `Seed` interface.

Reply:
Thanks done